WBooks/Library/Details/ReviewTable/ReviewViewController.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  ReviewViewController.swift
+//  WBooks
+//
+//  Created by ana.mancuso on 17/10/2022.
+//
+
+import UIKit
+
+class ReviewViewController: UIViewController {
+
+    //MARK: Properties
+    
+    private lazy var reviewView = ReviewView()
+    private let reviewModel: ReviewViewModel
```

Comment:
the name of the constant would be a little better if instead of Model it had ViewModel